OpenRA.Game/Traits/TraitsInterfaces.cs

```diff
@@ -266,6 +267,13 @@ public interface IRenderModifier
 		IEnumerable<Rectangle> ModifyScreenBounds(Actor self, WorldRenderer wr, IEnumerable<Rectangle> r);
 	}
 
+	[RequireExplicitImplementation]
+	public interface ICursorPaletteInfo : ITraitInfoInterface
```

Comment:
Sorry for continuously changing my mind on the naming here, but I now think we should use `IProvidesCursorPaletteInfo` for consistency with `IProvidesAssetBrowserPalettes`.